layer/manifest/XrApiLayer_dive.json

```diff
@@ -9,6 +9,6 @@
         "functions": {
             "xrNegotiateLoaderApiLayerInterface": "xrNegotiateLoaderApiLayerInterface"
         },
-        "disable_environment": "DISABLE_XR_APILAYER_DIVE"
+        "disable_environment": "DISABLE_XR_APILAYER_DIVE_1"
```

Comment:
out of curiosity, why `_1` here?

Reply:
do you know where is this `_1` coming from? is it because of `"api_version" : "1.0"`?